src/main/java/nextstep/subway/common/domain/Distance.java

```diff
@@ -0,0 +1,59 @@
+package nextstep.subway.common.domain;
+
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Distance {
+
+	private static final double ZERO = 0.0D;
+	private static final String COLUMN_DESCRIPTION = "역 사이 간격";
+
+	@Column
+	private double distance;
+
+	protected Distance() {
+	}
+
+	private Distance(double distance) {
+		validateDistance(distance);
+		this.distance = distance;
+	}
+
+	public static Distance generate(double distance) {
+		return new Distance(distance);
+	}
+
+	private void validateDistance(double distance) {
+		validateOverZero(distance);
+	}
```

Comment:
검증 메소드를 2개로 나누지 않아도 될 것 같네요.

```suggestion
	private void validateDistance(double distance) {
		if (distance <= ZERO) {
			throw new IllegalArgumentException(COLUMN_DESCRIPTION + "은 0을 초과하는 숫자여야 합니다.");
		}
	}
```

Reply:
검증 메소드가 여러개가 나올 것을 감안하고 작성한 것이었는데, 굳이 필요가 없을 듯합니다 😒 
해당 내용도 하나로 줄이도록 하겠습니다! 😄 
불필요한 코드 내용 알려주셔서 감사합니다!! 🙇 👍